install_minikube.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+source ./util.sh
+
+# inspired by https://github.com/LiliC/travis-minikube/blob/minikube-26-kube-1.10/.travis.yml
+
+export CHANGE_MINIKUBE_NONE_USER=true
+
+# install minikube if needed
+if hash minikube 2>/dev/null; then
+  echo "Skipping minikube install"
+else
+  echo "Installing minikube"
+  curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 && \
+    chmod +x minikube && sudo mv minikube /usr/local/bin/
+fi
+
+# Make root mounted as rshared to fix kube-dns issues.
+sudo mount --make-rshared /
+
+sudo minikube start --memory=8192 --cpus=4 \
+  --kubernetes-version=v1.10.0 \
+  --vm-driver=none \
+  --bootstrapper=localkube \
```

Comment:
any specific reason to use localkube? it's been deprecated and I don't think it will work with k8s 1.11

Reply:
it works. I also tried `kubeadm`, which [did not work](https://travis-ci.org/projectriff/fats/jobs/420890237).